tests/docker/Dockerfile

```diff
@@ -168,4 +168,8 @@ RUN mkdir -p /opt/scripts && \
     curl https://raw.githubusercontent.com/redpanda-data/seastar/2a9504b3238cba4150be59353bf8d0b3a01fe39c/scripts/seastar-addr2line -o /opt/scripts/seastar-addr2line && \
     chmod +x /opt/scripts/seastar-addr2line
 
+# Install remote_scripts
```

Comment:
In general I find dynamic solutions to be more complex than static. What is the benefit of using injection? The files already live in the same directories so the file management is the same, I copy it as the last docker action so rebuild is fast too the only difference is that with the injection we need to do additional action in the tests, the rest is the same.

Reply:
I'm not objecting to you deploying scripts separately for the new bits, but I'm asking that you don't change the existing ones -- it's pretty much orthogonal to the purpose of this PR, right?

As it is, this change breaks those tests in clustered ducktape until equivalent deployment changes are made there, and it increases the friction to working in that environment: right now, I can edit a script and re-run a test.  Without the runtime injection, I have to edit the script, then (assuming we add some ansible for deploying remote_scripts/), run ansible, then run the test.

I think there's a meaningful distinction between re-usable tools that will be shared between many tests (these are deployed as  part of the container image in docker ducktape or AMI in clustered ducktape), and scripts that are specific to a single test.  The latter type of script benefits from being live-injected, because it makes a faster edit/test cycle when creating a test.

Reply:
I've rolled back that change